templates/pdns-chroot-resolv.conf.j2

```diff
@@ -1,6 +1,8 @@
 # {{ ansible_managed }}
 
 [Service]
+# Cannot use Type=notify in chroot
+Type=simple
 PermissionsStartOnly=true
-ExecStartPre=/bin/install -dm755 "{{ pdns_auth_config.chroot }}/etc"
+ExecStartPre=/usr/bin/install -dm755 "{{ pdns_auth_config.chroot }}/etc"
```

Comment:
Does this path differ between distributions? You could set it depending on it.

Reply:
It's always /usr/bin, but on Arch it's symlinked and I was too lazy to type it